### ISSUE TITLE
Download feature doesn't seem to work and not documented

### ISSUE THREAD

**Issue by kwatsen** [moinwiki/moin#885]
Navigating to any page and clicking on the "Download" link, produces log output like the following:

```
[pid: 67896|app: 0|req: 2434/2434] 10.0.4.6 () {52 vars in 1320 bytes} [Mon Aug  5 13:37:36 2019] GET /+download/TestPage => generated 0 bytes in 79 msecs via sendfile() (HTTP/1.1 200) 10 headers in 784 bytes (1 switches on core 0)
```

The browser spins for awhile, ultimately failing saying "Network Error", and the resulting file on disk is 0 bytes in size.

None of this makes sense to me and, since there is no documentation, it is unclear what the expected behavior might be.

**Comment by RogerHaase** [comment]
Cannot reproduce on windows 10 using built-in server nor on https://moinmoin2.pythonanywhere.com/Home

I get normal file download dialog with save/open choice.

**Comment by kwatsen** [comment]
Confirmed, I also could download the "Home" page via the link provided, and now I see that the file contents are the moinmoin markup...

I tried again my server's Home and again it still doesn't work.  So the question is why not?  I suppose I should try again with a fresh install and see what happens.

BTW, does this project assign issues?   I'll see if I can assign this issue to me but, if not, maybe someone needs to do it for me?
